spec/requests/users_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'rails_helper'
 
-RSpec.describe "Users", type: :request do
-  describe "GET /users" do
-    it "works! (now write some real specs)" do
+RSpec.describe 'Users', type: :request do
+  describe 'GET /users' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.